### DISCUSSION TITLE
StreamApp changes (webrtcAppEE), how do I make them part of enterprise server deployed locally?

### DISCUSSION THREAD

**Discussion by hd51515** [ant-media/Ant-Media-Server#5372] [General]
I have to update streamapp for some experiments. I am wondering how do those changes reflect inside antm enterprise server.

**Comment by USAMAWIZARD** [comment]
for building your own application and deploying it you can follow these steps 

1. git clone https://github.com/ant-media/StreamApp
2. edit the redeply file in the root driectory of the folder and set the directory of AMS 
AMS_DIR=~/softwares/ant-media-server
3. make changes in the application 
3. run the redeploy script file it will install the application with your changes

**Reply by hd51515** [reply]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  13:35 min
[INFO] Finished at: 2023-07-20T00:03:01-04:00
[INFO] ------------------------------------------------------------------------
rm: cannot remove 'webapps/*App*': No such file or directory
bash: create_app.sh: No such file or directory
bash: create_app.sh: No such file or directory

**Reply by USAMAWIZARD** [reply]
did you set AMS_DIR to the path of your ant media directory 
AMS_DIR = /usr/local/antmedia

**Comment by hd51515** [comment]
Yes, it worked after setting AMS_DIR = /usr/local/antmedia
